library/std/src/Std/Math.qs

```diff
@@ -396,16 +396,27 @@ function Floor(value : Double) : Int {
 }
 
 /// # Summary
-/// Returns the nearest integer to the specified number.
-/// For example: Round(3.7) = 4; Round(-3.7) = -4
+/// Returns the nearest integer to the specified number. Half is rounded towards zero.
+/// For example: Round(3.7) = 4; Round(-3.7) = -4; Round(3.5) = 3;
+///
+/// # References
+/// [Wikipedia article - Rounding](https://en.wikipedia.org/wiki/Rounding#Rounding_half_toward_zero)
```

Comment:
Just curious what influenced the choice for the default behavior for the Round function to be to round half toward zero?

Reply:
discussed offline

Reply:
Old QDK behaved like that.